src/setup/devnet/templates/docker/docker-bor-setup.sh.njk

```diff
@@ -2,16 +2,21 @@
 
 set -x
 
+NODE_DIR=/root/var/lib/bor
+DATA_DIR=/root/var/lib/bor/data
+
+# # copy static nodes from json to toml
+# STATICNODES=$(cat $NODE_DIR/static-nodes.json)
+# sed -i "s%.*static-nodes =.*%static-nodes = $(echo $STATICNODES)%g" $NODE_DIR/config.toml
+
```

Comment:
We should be initializing the static nodes in the toml config right ?

Reply:
So I observed that we don't have to do it. I saw that even though the following logs were printed, the nodes were connected using the static-nodes.json file.
```
Found deprecated node list file /root/var/lib/bor/data/bor/static-nodes.json, please use the TOML config file instead.
The static-nodes.json file is deprecated and ignored. Use P2P.StaticNodes in config.toml instead.
```

Reply:
Oh nice, then we can remove the commented code. Or maybe add a reminder to uncomment once it's deprecated.

Reply:
Added a comment [here](https://github.com/maticnetwork/matic-cli/pull/221/commits/969b56c66137f3429c24edc54a08af1ad0d920a1).